### PR TITLE
ICMSLST-1837 Send email using Django SMTP mail backend.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Changes:
  - Support sending email via django / legacy code.
  - Add USE_LEGACY_EMAIL_CODE to swap email sending code.
  - Use common settings for all email sending code
  - Rename and group django environment variables

### CODE REVIEW

**Review by mikeylockwood — APPROVED** [review]

File: .circleci/ci.env [hunk 1]
```diff
@@ -4,13 +4,14 @@ DEBUG=True
...
-EMAIL_SMTP_PORT=1025
-EMAIL_USE_TLS=False
+# Mail settings (For both email sending functions)
+DJANGO_EMAIL_PORT=1025
+DJANGO_EMAIL_HOST=localhost
+DJANGO_EMAIL_HOST_USER=test_user
+DJANGO_EMAIL_HOST_PASSWORD=password
+DJANGO_EMAIL_USE_TLS=False
+
+USE_LEGACY_EMAIL_CODE=false
```
Comment (mikeylockwood) [inline .circleci/ci.env:14]: Should false be lowercase?

File: mail/chief/email.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # TODO: Replace this with https://uktrade.atlassian.net/browse/ICMSLST-1837
```
Comment (marcuspp) [inline mail/chief/email.py:1]: Remove TODO

**Review by marcuspp — APPROVED** [review]

File: .circleci/ci.env (hunk 1 above)
Comment (MattHolmes123) [reply .circleci/ci.env:14]: I will make them consistant, it supports either (as well as 0 or 1)

File: mail/chief/email.py (hunk 2 above)
Comment (MattHolmes123) [reply mail/chief/email.py:1]: Silly me :)

**Pull request merged**